core/actions/uploads.js

```diff
@@ -0,0 +1,20 @@
+export const FETCHING_UPLOAD_PROVIDERS = "FETCHING_UPLOAD_PROVIDERS";
+export const fetchingUploadProviders = () => ({
+  type: FETCHING_UPLOAD_PROVIDERS
+});
+
+export const FETCHING_UPLOAD_PROVIDERS_SUCCEEDED = "FETCHING_UPLOAD_PROVIDERS_SUCCEEDED";
+export const fetchingUploadProvidersSucceeded = providerSettings => ({
+  type: FETCHING_UPLOAD_PROVIDERS_SUCCEEDED,
+  payload: {
+    providerSettings
+  }
+});
+
+export const UPLOAD_FAILURE = "UPLOAD_FAILURE";
+export const uploadFailure = error => ({
```

Comment:
Do you use it anywhere?
I tried to cause an error with upload, putting invalid URL/credentials for the provider and I never saw the error message on the UI.
In addition I don't see the 'error' state var being used anywhere.

Reply:
We do not so I removed it for now.